Cmake/CMakeLists.txt

```diff
@@ -1,13 +1,18 @@
 ﻿# CMakeList.txt : CMake project for CMakeProject1, include source and define
 # project specific logic here.
-#
+# 
+
+project(My_Project)
```

Comment:
Change My_Project  to MyProject - so it is consistent with MyLib

Reply:
Fixed